tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/IdResolutionSpec.scala

```diff
@@ -118,5 +137,9 @@ class IdResolutionSpec extends BaseSpec {
     List(Accept(MediaRange.One(`text/html`, 1f)))
   private def fusionResourcePageFor(encodedId: String) =
     s"https://bbp.epfl.ch/nexus/web/$ref11/resources/$encodedId".replace("%3A", ":")
+  private def fusionResolveEndpoint(encodedId: String) =
+    s"https://bbp.epfl.ch/nexus/web/resolve/$encodedId".replace("%3A", ":")
+  private def deltaResolveEndpoint(encodedId: String)  =
+    s"http://delta:8080/v1/resolve/$encodedId".replace("%3A", ":")
```

Comment:
Shouldn't this be URL encoded?

Reply:
I'm not sure I follow why it should be encoded? I don't think the location in a redirect needs to be encoded?

Reply:
Only the last segment is encoded because it needs to be a single segment representing the resource ID, and that can contain `/`. Similar to how the id needs to be URL encoded when fetching resources, etc.

Reply:
I wonder if other unpredictable things could happen if url characters are in the ID